### PR TITLE
Fix elastic kibana deploy

### PR DESCRIPTION
**Pull request by JJediny**
Ready for review

### CODE REVIEW

File: ansible/kibana.yml [hunk 1]
```diff
@@ -7,3 +7,5 @@
 
   roles:
     - role: gsa.ansible-role-kibana
+      vars:
+        kibana_version: "6.6"
```
Comment (adborden) [inline ansible/kibana.yml:11]: Should this kind of variable be in the playbook or in the inventory?

File: ansible/inventories/mgmt/group_vars/all/vars.yml [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+# Elasticsearch/Kibana shared creds
+kibana_elasticsearch_username: "{{ vault_kibana_elasticsearch_username }}"
+kibana_elasticsearch_password: "{{ vault_kibana_elasticsearch_password }}"
+elasticsearch_name: "{{ vault_elasticsearch_name }}"
+elasticsearch_port: 9200
+kibana_port: 5601
+
+# Nginx basic auth
+basic_auth_user: "{{ vault_basic_auth_user }}"
+basic_auth_hash: "{{ vault_basic_auth_hash }}"
```
Comment (adborden) [inline ansible/inventories/mgmt/group_vars/all/vars.yml:14]: I didn't see these in the vault, maybe they are missing the `vault_` prefix in the vault?

**Review by adborden — APPROVED** [review]

File: ansible/kibana.yml (hunk 1 above)
Comment (JJediny) [reply ansible/kibana.yml:11]: I moved all other vars to inventory, but I wanted to make this declarative in the playbook to make it easier to upgrade and make it more obvious for the user deploying the role.

File: ansible/inventories/mgmt/group_vars/all/vars.yml (hunk 2 above)
Comment (JJediny) [reply ansible/inventories/mgmt/group_vars/all/vars.yml:14]: Good catch, added a commit to prepend the missing `vault_`

**Pull request merged**